### ISSUE TITLE
v2: Message field in response

### ISSUE THREAD

**Issue by dbrgn** [SpaceApi/validator#50]
In ValidateUrlV2Response, the `message` field is optional. In ValidateJsonV2Response, it's required (and will be returned as an empty string by default). Is there a reason for that?

**Comment by gidsi** [comment]
No, doesn't make any sense to me. I would assume that v1 send an empty string and i tried to comply with the v1 behavior (which seems kinda stupid in retrospective).

**Comment by dbrgn** [comment]
Ok, then the question is whether to always send a message or not. If it's required, that's easier for parsing (because you don't need to validate whether the field is there or not), but at the same time it's a bit stupid to send an empty message. I don't have a strong opinion on this :slightly_smiling_face:

**Comment by gidsi** [comment]
I would prefer to make it not required. I would assume that most people would use a lib for json parsing anyways.

**Comment by dbrgn** [comment]
Yep, but if you parse the JSON into a Python object and then call `response['message']` you still get an `IndexError`. But leaving it out is fine :slightly_smiling_face:
